crates/lang/ir/src/ir/item_impl/message.rs

```diff
@@ -100,6 +100,8 @@ pub struct Message {
     pub(super) item: syn::ImplItemMethod,
     /// If the ink! message can receive funds.
     is_payable: bool,
+    /// If the ink! message is a wildcard selector.
+    has_wildcard_selector: bool,
```

Comment:
We could ues `selector: Option<SelectorOrWildcard>` here instead of the current `has_wildcard_selector` and `selector` fields.